### ISSUE TITLE
'L.Polyline._flat is not a function' Error

### ISSUE THREAD

**Issue by Nimrod51** [IvanSanchez/Leaflet.Polyline.SnakeAnim#25]
Hi, even implementing this simple code results in this error: 
**'L.Polyline._flat is not a function**
Thus, only visualizing the line but not animating it. The only way the plugin would work was with your particular demo.html file. (Using Leaflet 1.1.0 on Chrome).

```
var latlngs = [
    [45.51, -122.68],
    [37.77, -122.43],
    [34.04, -118.2];

var polyline = L.polyline(latlngs, {color: 'red'});
polyline.addTo(map).snakeIn();
```

**Comment by Dahkon** [comment]
Got the same issue.
Replace L.Polyline._flat with L.LineUtil.isFlat.

**Comment by IvanSanchez** [comment]
Closed via #32
